pygenometracks/tracks/GtfTrack.py

```diff
@@ -0,0 +1,155 @@
+from . GenomeTrack import GenomeTrack
+from . BedTrack import BedTrack
+from .. readGtf import ReadGtf
+from matplotlib import font_manager
+import numpy as np
+
+DEFAULT_BED_COLOR = '#1f78b4'
+DISPLAY_BED_VALID = ['collapsed', 'triangles', 'interleaved', 'stacked']
+DISPLAY_BED_SYNONYMOUS = {'interlaced': 'interleaved', 'domain': 'interleaved'}
+DEFAULT_DISPLAY_BED = 'stacked'
+
+
+class GtfTrack(BedTrack):
+    SUPPORTED_ENDINGS = ['gtf', 'gtf.gz']
```

Comment:
does it matter? I thought you wanted to check for the format of the file based on the given data type and check if the format is right regardless of the ending. Did I get you wrong?

Reply:
The SUPPORTED_ENDINGS is used by make_tracks_file